recipe/meta.yaml

```diff
@@ -1,50 +1,45 @@
 {% set name = "openai" %}
-{% set version = "0.27.4" %}
+{% set version = "1.3.6" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/openai/openai-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 5a144dc655d6f097503c49625ddc423b28f676b96e51679093c5489a997c7f6a
+  sha256: 47e856250e456fafb3368166e895647dc38aae998d0286473cfad2bb78a932e3
 
 build:
-  number: 1
+  number: 0
   entry_points:
-    - openai=openai._openai_scripts:main
+    - openai=openai.cli:main
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   # Minimum required version is 3.7.1, and s390x does not have pandas-stubs
   skip: True # [py<37 or s390x]
 
 requirements:
   host:
-    - pip
-    - wheel
     - python
-    - setuptools
+    - pip
+    - hatchling
   run:
-    - aiohttp
     - python
-    - requests >=2.20
-    - tqdm
-    - colorama
-    - typing_extensions  # [py<38]
+    - anyio <4,>=3.5.0
+    - distro <2,>=1.7.0
+    - httpx <1,>=0.23.0
+    - pydantic <3,>=1.9.0
+    - sniffio
+    - tqdm >4
+    - typing-extensions <5,>=4.5
     # The following are required for the optional things.
```

Comment:
if these are optional, shouldn't they go to `run_constrained`?
https://github.com/openai/openai-python/blob/17ac6779958b2b74999c634c4ea4c7b74906027a/pyproject.toml#L39

Reply:
It would technically be a breaking change to our users... We can but we need to consider the impact.

Reply:
numpy is used to return [embedding vectors](https://en.wikipedia.org/wiki/Word_embedding) in base64 format: https://github.com/openai/openai-python/blob/17ac6779958b2b74999c634c4ea4c7b74906027a/src/openai/resources/embeddings.py#L85

pandas is used to validate data (before sending it to the API maybe?) https://github.com/openai/openai-python/blob/17ac6779958b2b74999c634c4ea4c7b74906027a/src/openai/lib/_validators.py#L10

this package isn't used as a dependency much. Maybe putting these as run_constrained would be better to improve co-installability? I don't think it matters a great deal either way though.

Reply:
I think I'll leave them there for now.